ci/terraform/integration.tfvars

```diff
@@ -5,10 +5,11 @@ frontend_auto_scaling_enabled   = true
 frontend_task_definition_cpu    = 512
 frontend_task_definition_memory = 1024
 
-support_international_numbers = "1"
-support_language_cy           = "1"
-support_account_recovery      = "1"
-support_smart_agent           = "0"
+support_welsh_language_in_support_forms = "0"
```

Comment:
Is this disabled intentionally?

Reply:
Thanks @JHjava - I'd initially thought it would be good to enable it in Build only but it makes sense to do it in Integration too. I've just amended the commit to set this to `"1"`